src/main/java/com/autotune/analyzer/experiment/ExperimentInitiator.java

```diff
@@ -191,7 +191,10 @@ public void validateAndAddExperimentResults(List<UpdateResultsAPIObject> updateR
         List<ExperimentResultData> resultDataList = new ArrayList<>();
         successUpdateResultsAPIObjects.forEach(
                 (successObj) -> {
-                    resultDataList.add(Converters.KruizeObjectConverters.convertUpdateResultsAPIObjToExperimentResultData(successObj));
+                    ExperimentResultData experimentResultData = Converters.KruizeObjectConverters.convertUpdateResultsAPIObjToExperimentResultData(successObj);
```

Comment:
experimentResultData.getValidationOutputData().isSuccess() 
is not success then how we are going to report to user 
is it getting added to 
failedUpdateResultsAPIObjects  ?

Reply:
To test this secanrio try throw Exception manually and see if it is reported back to response

Reply:
I tried adding multiple objects and failed a couple of them to check the response. Here's what I'm getting:

```
{
    "message": "Out of a total of 3 records, 2 failed to save",
    "httpcode": 400,
    "documentationLink": "",
    "status": "ERROR",
    "data": [
        {
            "interval_start_time": "2023-09-10T14:00:00.000Z",
            "interval_end_time": "2023-09-10T14:15:00.000Z",
            "errors": [
                {
                    "message": "kubernetes_objects : Metric data is not present for container : tfb-server-0 for experiment: 5kexp_1",
                    "httpcode": 400,
                    "documentationLink": "",
                    "status": "ERROR"
                }
            ],
            "version": "1.0",
            "experiment_name": "5kexp_1"
        },
        {
            "interval_start_time": "2023-09-10T15:30:00.000Z",
            "interval_end_time": "2023-09-10T15:45:00.000Z",
            "errors": [
                {
                    "message": "kubernetes_objects : Metric data is not present for container : tfb-server-0 for experiment: 5kexp_1",
                    "httpcode": 400,
                    "documentationLink": "",
                    "status": "ERROR"
                }
            ],
            "version": "1.0",
            "experiment_name": "5kexp_1"
        }
    ]
}
```